.github/workflows/spherical_geometry.yml

```diff
@@ -73,3 +76,21 @@ jobs:
       uses: codecov/codecov-action@v1
       with:
         file: ./coverage.xml
+
+  bandit:
+    name: Bandit
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install bandit
+    - name: Security checks with bandit
+      run: |
+        bandit romancal -r -x tests
```

Comment:
`romancal` means this is not checking anything.

Reply:
Corrected this.